### PR TITLE
user guides: Review user documentation for Organization Basics, Part 1.

### PR DESCRIPTION
**Pull request by ViRu-ThE-ViRuS**
GCI Task -> https://codein.withgoogle.com/dashboard/task-instances/5969345714847744/

### CODE REVIEW

**Comment by shubhamdhama** [comment]
Nice work, @ViRu-ThE-ViRuS , though there are some typos and some grammatical mistakes which if removed will make this note awesome. Leaving some comments. Though Nice read 👌.

File: user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Tools and Customization
+
+### Review User Guides On:
+ - Change your organization name
+ - Change your organization description
+ - Change your organization avatar
+ - Change your organization's settings
+ - Link to our Zulip from the web
+
+The procedures and user experience for all the above operations is pretty straight forward.
```
Comment (shubhamdhama) [inline user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md]: `is`->`are` and `straight forward`-> `straightforward`

File: user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+The key is then simplicity: we don't want to over complicate the steps for a simple task.
+I did all the listed operations on a local dev instance in MacOS High Sierra, by following all the guides word by word.
+
+### Breakdown:
+#### Change your organization name
+[Link.](https://zulipchat.com/help/change-your-organizations-name)
+The steps for this operation are easy to follow and short. I only recommend to show how to access the `Manage Organization` page in this page itself. It currently requires to navigate to a whole new page.
+
```
Comment (shubhamdhama) [inline user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md]: `to show` -> `showing` and `to navigate` -> `navigating`

File: user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#### Change your organization description
+[Link.](https://zulipchat.com/help/change-your-organizations-description)
+Same recommendation as the last one. A short description on how to access the `Manage Organization` page will save the daily user a click. The steps otherwise are concise and accurate.
+
+
+#### Change your organization avatar
+[Link.](https://zulipchat.com/help/change-your-organizations-avatar)
+Same recommendation as the last one. The steps otherwise are easy to follow and I dont have any questions. It would be nice to specify all if any specifications about the avatar image like size and format.
+
```
Comment (shubhamdhama) [inline user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md]: `dont`->`don't`

File: user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+#### Change your organization's settings
+[Link.](https://zulipchat.com/help/change-your-organization-settings)
+The `Manage Organization` option in the dropdown menu from step 1 is called the `Organization settings` in the current version of Zulip. This needs to updated as a user might get confused. Step 3 is not necessarily a step as such.
+
```
Comment (shubhamdhama) [inline user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md]: `updated` ->`update`

File: user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+#### Link to your Zulip from the web
+[Link.](https://zulipchat.com/help/join-zulip-chat-badge)
+The documentation for the operation is short, concise and easy to follow. I have no questions/suggestions.
+
+### Conclusion:
+The documentations for the listed operations were easy to follow and concise. The minor suggestions I make may potentially improve the user experience. The updation of `Manage Organization` tab to `Organization settings` tab is quite necessary though.
```
Comment (shubhamdhama) [inline user-guides/ViRu-ThE-ViRuS/organization-basics-part1.md]: `updation of`->`need to update` (as updation is not a word)

**Comment by ViRu-ThE-ViRuS** [comment]
@shubhamdhama Thank you sir for helping me fix my ignorance towards grammar. I will make the requested changes and submit shortly.

**Comment by ViRu-ThE-ViRuS** [comment]
@shubhamdhama I have made all the suggested changes sir.

**Comment by shubhamdhama** [comment]
Now it's looking fine, nice work though.

**Pull request merged**